spring_jdbc/src/main/java/com/geekhub/dao/impl/TyresDaoImpl.java

```diff
@@ -0,0 +1,68 @@
+package com.geekhub.dao.impl;
+
+import com.geekhub.dao.TyresDao;
+import com.geekhub.dao.mapper.TyresMapper;
+import com.geekhub.model.Tyres;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.support.JdbcDaoSupport;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.PostConstruct;
+import javax.sql.DataSource;
+import java.util.List;
+
+@Service
+@Transactional
+public class TyresDaoImpl extends JdbcDaoSupport implements TyresDao {
+    @Autowired
+    private DataSource dataSource;
+
+    @PostConstruct
+    private void initialize() {
+        setDataSource(dataSource);
+    }
+
+    @Override
+    public void saveTyres(Tyres tyres) {
+        try {
+            this.getJdbcTemplate().update
+                    ("INSERT INTO Tyres (SizeTyres, NameTyres) VALUES(?,?)",
+                            new Object[] { tyres.getSize(), tyres.getName() });
+        }catch (Exception e){
```

Comment:
This is not recommended to catch (Exception) and then just log error. This way you hide the error from external classes. For instance the transactions will not be rolled back.